blockade/state.py

```diff
@@ -186,3 +186,12 @@ def __write(self, containers, initialize=True):
             # clean up our created file
             self._state_delete()
             raise
+
+    def get_audit_file(self):
+        audit_dir = os.path.join(self._state_dir, "audit")
+        try:
+            os.makedirs(audit_dir, 0o755)
+        except OSError as os_e:
+            if os_e.errno != 17:
+                raise
+        return os.path.join(audit_dir, "%s.json" % self._blockade_id)
```

Comment:
It was not clear to me that was the case for daemon mode or for when --config was used.